src/pages/IndexPage.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <q-page class="column items-center justify-evenly">
+    <div>test1 début</div>
+    <HelloWorld/>
+    <div>test1 fin</div>
+  </q-page>
+</template>
+
+<script setup lang="ts">
+import {defineAsyncComponent} from 'vue';
+
+const HelloWorld = defineAsyncComponent(() => import('catalog-ui/HelloWorld'));
```

Comment:
There is no error handling if the remote module fails

Reply:
yes, it will be replace by corelib function in another task (those with the lifecyle).